samples/scheduler/program.py

```diff
@@ -0,0 +1,66 @@
+"""
+This program demonstrates AutoKitteh's scheduler capabilities.
+
+It implements a single entry-point function, configured in the "autokitteh.yaml"
+file to receive "scheduler" events, and uses constant values defined in the
+"autokitteh.yaml" manifest for each AutoKitteh environment.
+"""
+
+from datetime import datetime, timedelta, timezone
+import os
+from autokitteh.github import github_client
+from autokitteh.slack import slack_client
+
+# Set in "autokitteh.yaml"
+GITHUB_OWNER = os.getenv("GITHUB_OWNER")
+GITHUB_REPO = os.getenv("GITHUB_REPO")
+SLACK_CHANNEL_NAME_OR_ID = os.getenv("SLACK_CHANNEL_NAME_OR_ID")
+
+
+def on_cron_trigger(_):
+    """Handles the AutoKitteh cron schedule trigger."""
+
+    github = github_client("github_conn")
+    slack = slack_client("slack_conn")
+
+    # Fetch open pull requests that are not drafts or WIP
+    repo = github.get_repo(f"{GITHUB_OWNER}/{GITHUB_REPO}")
+    active_prs = [
+        pr
+        for pr in repo.get_pulls(state="open")
+        if not pr.draft
+        and "draft" not in pr.title.lower()
+        and "wip" not in pr.title.lower()
+    ]
+
+    now = datetime.now(timezone.utc)
+    opened_cutoff = now - timedelta(days=4)
+    update_cutoff = now - timedelta(days=1)
```

Comment:
the number of days need to be configurable

Reply:
Done